extensions/blocks/publicize/twitter/tweet-divider.js

```diff
@@ -0,0 +1,324 @@
+		if ( ! supportedBlock ) {
+			popoverWarnings.push( __( 'This block is not exportable to Twitter', 'jetpack' ) );
+		} else {
+			if ( 'core/gallery' === childProps.name && childProps.attributes.images.length > 4 ) {
+				popoverWarnings.push( __( 'Twitter displays the first four images.', 'jetpack' ) );
+			}
+
+			if ( findTagsInContent( [ 'strong', 'bold', 'em', 'i', 'sup', 'sub', 'span', 's' ] ) ) {
+				popoverWarnings.push( __( 'Twitter removes all text formatting.', 'jetpack' ) );
+			}
+
+			if ( findTagsInContent( [ 'a' ] ) ) {
+				popoverWarnings.push( __( 'Links will be posted seperately.', 'jetpack' ) );
+			}
+		}
+
+		// Don't show the popover when the user is clearly doing something else.
+		const shouldShowPopover =
+			! isTyping() &&
+			! isDraggingBlocks() &&
+			! isMultiSelecting() &&
+			! hasMultiSelection() &&
+			! isCaretWithinFormattedText() &&
+			popoverWarnings.length > 0;
+
+		return {
+			isTweetstorm,
+			isSelectedTweetBoundary,
+			contentAttributesChanged,
+			boundaries,
+			blockStyles,
+			popoverWarnings,
+			shouldShowPopover,
+			currentAnnotations,
+		};
+	} ),
+	withDispatch( ( dispatch, { childProps, contentAttributesChanged }, { select } ) => {
+		return {
+			updateTweets: () => dispatch( 'jetpack/publicize' ).refreshTweets(),
+			updateAnnotations: () => {
+				// If this block hasn't been assigned to a tweet, skip annotation work.
+				const tweets = select( 'jetpack/publicize' ).getTweetsForBlock( childProps.clientId );
+				if ( ! tweets || tweets.length === 0 ) {
+					return;
+				}
+
+				// Check if the block content has changed since we sent it to the server for analysis.
+				// If it has changed, don't update annotations, since it's better to leave them in the
+				// same place, (even if that's incorrect), instead of moving them to a place where they
+				// were correct a few seconds ago, but may be incorrect now.
+				const blockCopy = tweets.reduce( ( foundBlock, tweet ) => {
+					if ( foundBlock ) {
+						return foundBlock;
+					}
+
+					return tweet.blocks.find( block => block.clientId === childProps.clientId );
+				}, false );
+
+				if ( contentAttributesChanged( blockCopy.attributes, childProps.attributes ) ) {
+					return;
+				}
+
+				// Remove any existing annotations in this block.
+				const annotations = select( 'core/annotations' ).__experimentalGetAllAnnotationsForBlock(
+					childProps.clientId
+				);
+				annotations.forEach( annotation => {
+					if ( annotation.source === 'jetpack-tweetstorm' ) {
+						dispatch( 'core/annotations' ).__experimentalRemoveAnnotation( annotation.id );
+					}
+				} );
+
+				const boundaries = tweets.filter( tweet => tweet.boundary ).map( tweet => tweet.boundary );
+
+				// Add new annotations in the appropriate location.
+				boundaries.forEach( boundary => {
+					const { container, type, start, end } = boundary;
+					if ( 'normal' === type ) {
+						dispatch( 'core/annotations' ).__experimentalAddAnnotation( {
```

Comment:
For my own reference and for anyone who may be asking themselves the same thing; although experimental, those were already available in WordPress 5.4.